### PR TITLE
add transaction entity

### LINKED ISSUE #18: [Transaction]  Create transaction entity with attributes and require classes

### PR DESCRIPTION
**Pull request by VoHoangAn**
first time... huh ?

### CODE REVIEW

File: src/main/java/com/fjb/sunrise/models/Transaction.java [hunk 1]
```diff
@@ -1,28 +1,54 @@
...
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 
 @Entity
 @Getter
 @Setter
-@Builder
```
Comment (khanhduzz) [inline src/main/java/com/fjb/sunrise/models/Transaction.java:17]: consider keep it for later use

File: src/main/java/com/fjb/sunrise/models/Transaction.java [hunk 2]
```diff
@@ -1,28 +1,54 @@
 package com.fjb.sunrise.models;
 
+import com.fjb.sunrise.enums.ETrans;
+import jakarta.persistence.CascadeType;
```
Comment (khanhduzz) [inline src/main/java/com/fjb/sunrise/models/Transaction.java]: remove redundant import when not use for cleaner code

**Comment by khanhduzz** [comment]
For naming, try to name the PR in the clear way
For ex:
[Transaction] Create entity ...

**Review by khanhduzz — APPROVED** [review]

**Review by NhatTranMinh15 — APPROVED** [review]

**Review by QuyDang1108 — APPROVED** [review]
so good :v

**Pull request merged**